__tests__/test-utils/mock-data/manager.js

```diff
@@ -13,6 +13,23 @@ import type {
 
 let COUNTER = 0
 
+/**
+ * Make a mock deployment given a project and some FeedVersions. This is a
```

Comment:
I suppose you could say 
```suggestion
 * Make a mock deployment summary given a project and some FeedVersions. This is a
```

Reply:
Good catch thank you! Adjusted.